exporters/variables-scss/src/content/generators/__tests__/contentGenerator.test.ts

```diff
@@ -0,0 +1,69 @@
+import fs from 'fs';
+import path from 'path';
+import { Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
+import { generateFileContent, addDisclaimer, filterTokensByTypeAndGroup } from '../contentGenerator';
+import { exampleMockedGroups, exampleMockedTokens } from '../../__fixtures__/mockedExampleTokens';
+
+const mockedExpectedResult = fs.readFileSync(
+  path.join(__dirname, '../../__fixtures__/exampleFileContent.scss'),
+  'utf-8',
+);
+const mappedTokens: Map<string, Token> = new Map([]);
+const tokenGroups: Array<TokenGroup> = exampleMockedGroups;
+
+describe('contentGenerator', () => {
+  describe('generateFileContent', () => {
+    it('should generate file content', () => {
+      const tokens = Array.from(exampleMockedTokens.values());
+      const tokenTypes = [TokenType.dimension, TokenType.string];
+      const groupNames = ['Grid', 'String'];
+      const withCssObject = true;
+      const hasParentPrefix = true;
+
+      const fileContent = generateFileContent(
```

Comment:
Are you aware that this test also calls underlying functions used in `generateFileContent`. So this is some kind of integration test. There is a way to test only the function itself, it is called mocking. You can mock all those underlying functions and aim only at the function itself. And then it will be a unit test.

I do not want to refactor it now. I am glad that the function is tested. Thanks :-)

Reply:
I know mocking, but I don't think it's necessary here, right? This was apparently from CR to the still unfinished test implementation. Now all functions are covered by tests.